test/e2e/suite/issuers/ca/clusterissuer.go

```diff
@@ -67,3 +68,4 @@ var _ = framework.CertManagerDescribe("CA ClusterIssuer", func() {
 		Expect(err).NotTo(HaveOccurred())
 	})
 })
+*/
```

Comment:
Disabled the ClusterIssuer tests for now as we need to run these with a global instance!